projects/dxilconv/lib/DxilConvPasses/ScopeNestedCFG.cpp

```diff
@@ -548,7 +548,8 @@ void ScopeNestedCFG::AnnotateBranch(BasicBlock *pBB, BranchKind Kind) {
     ConstantAsMetadata *p1 = dyn_cast<ConstantAsMetadata>(pMD->getOperand(0));
     ConstantInt *pVal = dyn_cast<ConstantInt>(p1->getValue());
     BranchKind OldKind = (BranchKind)pVal->getZExtValue();
-    DXASSERT(OldKind == Kind || 
+    (void)(OldKind);
+    DXASSERT(OldKind == Kind ||
```

Comment:
Here on the other hand you can use DXASSERT_LOCALVAR() to add the (void) cast. For some reason DXASSERT doesn't add it by default.